README.md

```diff
@@ -113,11 +113,27 @@ pnpm nx build dialtone-documentation
 
 ### Releasing
 
+Running this commands will automatically release all packages that need to be released.
+
+#### Production
+
+Note: needs to be run while on staging branch.
+
+```bash
+pnpm run release
+```
+
+#### Alpha/Beta
+
+Note: needs to be run while on your feature branch.
```

Comment:
For this one, detail how it:

Merges the feature branch you are currently on to the "alpha" or "beta" branch
publishes the library package to npm under the @alpha or @beta distribution tag.